test/sql-luatest/gh_7283_result_parameter_type_test.lua 

```diff
@@ -0,0 +1,28 @@
+
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function()
+    g.server = server:new({alias = 'result_parameter_type'})
+    g.server:start()
+end)
+
+g.after_all(function()
+    g.server:stop()
+end)
+
+g.test_result_parameter_type_1 = function()
+    g.server:exec(function()
+        local t = require('luatest')
```

Comment:
`Luatest` is already available on your server instance.
Feel free to use it `t.assert(...)` 👍